webtau-cli-groovy/src/test/groovy/com/twosigma/webtau/cli/CliGroovyTest.groovy

```diff
@@ -31,7 +31,7 @@ class CliGroovyTest {
 
                 output.should == ~/hello/
                 output.should contain('world')
-                output.should contain('"message to world"')
+                output.should contain('message to world')
```

Comment:
I believe the previous behaviour was actually incorrect.  Invoking the same command from a terminal did not result in quoted message to world:
```bash
$ scripts/hello "message to world"
hello world message to world
line in the middle
more text
error line one
error line two
```